Cargo.toml

```diff
@@ -58,6 +58,9 @@ gcp-oidc = ["dep:reqwest"]
 # This can only be used with the tokio-runtime feature flag.
```

Comment:
@abr-egn

Unrelated to this PR, really (though we could fix it here), should we update this comment since async-std support was dropped?

Reply:
Sorry, yup, just delete the comment.  The driver only supports tokio now so the there's nothing that needs to be documented about this feature.